2022-03-06 11:58:56,683 INFO    MainThread:695893 [wandb_setup.py:_flush():71] setting env: {}
2022-03-06 11:58:56,683 INFO    MainThread:695893 [wandb_setup.py:_flush():71] setting login settings: {}
2022-03-06 11:58:56,683 INFO    MainThread:695893 [wandb_setup.py:_flush():71] setting login settings: {}
2022-03-06 11:58:56,684 INFO    MainThread:695893 [wandb_init.py:_log_setup():371] Logging user logs to /home/junshick/Workspace/pose_estimation/custom_pose/wandb/run-20220306_115856-2vmzz7ch/logs/debug.log
2022-03-06 11:58:56,684 INFO    MainThread:695893 [wandb_init.py:_log_setup():372] Logging internal logs to /home/junshick/Workspace/pose_estimation/custom_pose/wandb/run-20220306_115856-2vmzz7ch/logs/debug-internal.log
2022-03-06 11:58:56,684 INFO    MainThread:695893 [wandb_init.py:init():404] calling init triggers
2022-03-06 11:58:56,684 INFO    MainThread:695893 [wandb_init.py:init():411] wandb.init called with sweep_config: {}
config: {}
2022-03-06 11:58:56,684 INFO    MainThread:695893 [wandb_init.py:init():460] starting backend
2022-03-06 11:58:56,684 INFO    MainThread:695893 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-03-06 11:58:56,690 INFO    MainThread:695893 [backend.py:ensure_launched():216] starting backend process...
2022-03-06 11:58:56,694 INFO    MainThread:695893 [backend.py:ensure_launched():222] started backend process with pid: 695902
2022-03-06 11:58:56,695 INFO    MainThread:695893 [wandb_init.py:init():469] backend started and connected
2022-03-06 11:58:56,696 INFO    MainThread:695893 [wandb_init.py:init():533] updated telemetry
2022-03-06 11:58:56,707 INFO    MainThread:695893 [wandb_init.py:init():563] communicating current version
2022-03-06 11:58:57,766 INFO    MainThread:695893 [wandb_init.py:init():568] got version response upgrade_message: "wandb version 0.12.11 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-03-06 11:58:57,766 INFO    MainThread:695893 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-03-06 11:58:58,322 INFO    MainThread:695893 [wandb_init.py:init():606] starting run threads in backend
2022-03-06 11:59:00,968 INFO    MainThread:695893 [wandb_run.py:_console_start():1810] atexit reg
2022-03-06 11:59:00,969 INFO    MainThread:695893 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-03-06 11:59:00,969 INFO    MainThread:695893 [wandb_run.py:_redirect():1689] Redirecting console.
2022-03-06 11:59:00,970 INFO    MainThread:695893 [wandb_run.py:_redirect():1745] Redirects installed.
2022-03-06 11:59:00,971 INFO    MainThread:695893 [wandb_init.py:init():633] run started, returning control to user process
2022-03-06 11:59:03,167 INFO    MainThread:695893 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 1
2022-03-06 11:59:03,168 INFO    MainThread:695893 [wandb_run.py:_restore():1752] restore
2022-03-06 11:59:05,983 INFO    MainThread:695893 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 766
  total_bytes: 766
}

2022-03-06 11:59:06,336 INFO    MainThread:695893 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 766
  total_bytes: 766
}

2022-03-06 11:59:07,333 INFO    MainThread:695893 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 766
  total_bytes: 5982
}

2022-03-06 11:59:07,434 INFO    MainThread:695893 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 766
  total_bytes: 15581
}

2022-03-06 11:59:07,537 INFO    MainThread:695893 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 766
  total_bytes: 15581
}

2022-03-06 11:59:07,639 INFO    MainThread:695893 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 766
  total_bytes: 15581
}

2022-03-06 11:59:07,741 INFO    MainThread:695893 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15581
  total_bytes: 15581
}

2022-03-06 11:59:07,843 INFO    MainThread:695893 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15581
  total_bytes: 15581
}

2022-03-06 11:59:07,945 INFO    MainThread:695893 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15581
  total_bytes: 15581
}

2022-03-06 11:59:08,047 INFO    MainThread:695893 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15581
  total_bytes: 15581
}

2022-03-06 11:59:08,150 INFO    MainThread:695893 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15581
  total_bytes: 15581
}

2022-03-06 11:59:08,545 INFO    MainThread:695893 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15581
  total_bytes: 15581
}

2022-03-06 11:59:09,218 INFO    MainThread:695893 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15581
  total_bytes: 15581
}
local_info {
}

2022-03-06 11:59:10,283 INFO    MainThread:695893 [wandb_run.py:_append_files():2180] logging synced files
